packages/server/modules/workspaces/repositories/workspaces.ts

```diff
@@ -168,7 +168,7 @@ export const upsertWorkspaceRoleFactory =
 
 export const getWorkspaceCollaboratorsFactory =
   ({ db }: { db: Knex }): GetWorkspaceCollaborators =>
-  async ({ workspaceId, filter = {} }) => {
+  async ({ workspaceId, filter = {}, cursor, limit }) => {
```

Comment:
1) There should be a default limit, otherwise the query is just gonna return everything in one long list
2) The limit, whatever it ends up being, should be clamped (lodash's `clamp()`) between an allowed minimum and maximum, lets say 0 and 100